wagtail/images/templates/wagtailimages/bulk_actions/confirm_bulk_add_tags.html

```diff
@@ -0,0 +1,65 @@
+{% extends 'wagtailadmin/bulk_actions/confirmation/base.html' %}
+{% load i18n %}
+{% load wagtailimages_tags wagtailadmin_tags %}
+{% block titletag %}{% blocktrans count counter=images|length %}Add tags to 1 image {% plural %}Add tags to {{ counter }} images{% endblocktrans %}{% endblock %}
+
+{% block extra_js %}
+    {{ block.super }}
+
+    {{ form.media.js }}
```

Comment:
suggestion:
add the `extra_js` and `extra_css` blocks to `wagtailadmin/bulk_actions/confirmation/base.html` and include `form.media.js`/`form.media.css` in them at the base level.